devenv/README.md

```diff
@@ -0,0 +1,137 @@
+# Docker Based Development Environment
+
+This is a collection of Dockerized services to provide a simple 
+standalone development environment to sBTC developers. It also 
+includes some helper scripts to make it easier to operate.
+
+## Docker and Docker Compose
+
+To use this environment, you must install:
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+## Building Containers
+
+In order to deploy the environment, you must first build the images.
+
+```
+cd devenv 
+./build.sh
+```
+
+If you prefer only to build a specific container:
+
+```
+cd devenv/bitcoin/
+./build.sh
+```
+
+## Running Containers
+
+To start the entire deployment simply run:
+
+```
+cd devenv
+./up.sh
+```
+
+By default, this will start a BTC node on regtest, 
+STX node on mocknet, stacks-api and database.
+
+If you prefer to run a specific container:
+
+```
+cd devenv/bitcoin/
+./up.sh
+```
+
+## Logging Containers
+
+There is a helper script at the top level directory to facilitate logging:
+
+```
+./log bitcoin
+./log stacks
+./log stacks-api
+./log stacks-explorer
+./log postgres
+```
+
+## Mining
+
+For a development environment we need to control block production, 
+there is a helper script to do this:
+
+```
+cd devenv/
+./mine.sh <number of blocks>
+```
+Once you mine blocks on bitcoin, there is a delay before the 
+stacks mines those blocks, and relays the data to the API. If
+your explorer is crashing, just wait a few minutes and try again.
+
+## Services
```

Comment:
are there any default accounts for the services or everyone is open access without accounts?